packages/backend-core/src/features/index.ts

```diff
@@ -236,6 +252,11 @@ export class FlagSet<V extends Flag<any>, T extends { [key: string]: V }> {
         }
       }
 
+      if (requestContext) {
+        requestContext.featureFlagCache ??= {}
+        requestContext.featureFlagCache[this.setId] = flagValues
```

Comment:
Normally we would have getters/setters in the context rather than directly utilising the context - makes it a bit easier to use for people unfamiliar with it.

While we do usually do the setting via `doIn...` type functions but something to just set in the context works fine as well, like `context.setFeatureFlags()` and `context.getFeatureFlags()`

Reply:
Addressed 🙂